packages/app/src/cli/services/web-env.ts

```diff
@@ -0,0 +1,101 @@
+import {selectApp} from './dev/select-app.js'
+import {AppInterface} from '../models/app/app.js'
+import {output, file} from '@shopify/cli-kit'
+
+export type Format = 'json' | 'text'
+interface WebEnvOptions {
+  update: boolean
+  envFile: string
+}
+
+export async function webEnv(app: AppInterface, {update, envFile}: WebEnvOptions): Promise<output.Message> {
+  if (update) {
+    return updateEnvFile(app, {envFile})
+  } else {
+    return outputEnv(app)
+  }
+}
+
+export async function updateEnvFile(
+  app: AppInterface,
+  {envFile}: Pick<WebEnvOptions, 'envFile'>,
+): Promise<output.Message> {
+  let envFileContent = null
+
+  if (await file.exists(envFile)) {
+    envFileContent = await file.read(envFile)
+
+    output.info(`Current ${envFile} is:
+${envFileContent}
+`)
+  } else {
+    output.info(`No environment file found. Creating ${envFile}`)
+  }
+
+  const selectedApp = await selectApp()
+
+  const updatedValues = {
+    SHOPIFY_API_KEY: selectedApp.apiKey,
+    SHOPIFY_API_SECRET: selectedApp.apiSecretKeys[0].secret,
+    SCOPES: app.configuration.scopes,
+  }
+
+  const newEnvFileContent = patchEnvFile(envFileContent, updatedValues)
+  await file.write(envFile, newEnvFileContent)
+
+  return output.content`Updated ${envFile} to be:
+${newEnvFileContent}
+`
+}
+
+export async function outputEnv(app: AppInterface): Promise<output.Message> {
```

Comment:
I might refactor this to accept a `format` param so that I can reuse the same logic in the `info --web-dev` command.

Reply:
Yes, it would essentially be a redirect to the new command. Not sure what's our policy here but I can imagine people having already used the `--web-dev` command to produce their own `.env` files with a custom script.